### ISSUE TITLE
Include support for SEUS PTGI HRR 3.0

### ISSUE THREAD

**Issue by JustReghan** [IrisShaders/Iris#1133]
I know that not too long ago support for version 2.1 was added but since 3.0 has been released could iris include support for that version too?

For now I'll keep using 2.1

The new SEUS PTGI version can be found here: https://www.patreon.com/posts/download-seus-3-60268558

**Comment by MrX13415** [comment]
Here is the log when trying to load the HRR 3.0 shader.

```
23:05:14.269
Destroying pipeline OVERWORLD
23:05:14.612
Error while parsing the block ID map entry "heavy_weighted_pressure_plate:powered=false":
23:05:14.612
- The block minecraft:heavy_weighted_pressure_plate has no property with the name powered, ignoring!
23:05:14.612
Error while parsing the block ID map entry "heavy_weighted_pressure_plate:powered=true":
23:05:14.612
- The block minecraft:heavy_weighted_pressure_plate has no property with the name powered, ignoring!
23:05:14.621
Using shaderpack: SEUS PTGI HRR 3.zip
23:05:14.648
Creating pipeline OVERWORLD
23:05:14.889
[Triforce Patcher] gl_FragColor is not supported yet, please use gl_FragData! Assuming that the shaderpack author intended to use gl_FragData[0]...
23:05:14.892
Program link log for composite5: Fragment info
-------------
0(5772) : warning C7050: "col" might be used before being initialized
23:05:14.923
[Triforce Patcher] gl_FragColor is not supported yet, please use gl_FragData! Assuming that the shaderpack author intended to use gl_FragData[0]...
23:05:14.947
Couldn't load NewWorldRenderingPipeline, falling back to vanilla shaders.
net.minecraft.class_2973: Invalid shaders/core/gbuffers_sky_basic.json: Couldn't compile fragment program (<iris shaderpack shaders>, gbuffers_sky_basic) : 0(3159) : error C1503: undefined variable "MC_RENDER_STAGE_STARS"
	at net.minecraft.class_2973.method_12856(class_2973.java:48)
	at net.minecraft.class_5944.<init>(class_5944.java:200)
	at net.coderbot.iris.pipeline.newshader.ExtendedShader.<init>(ExtendedShader.java:32)
	at net.coderbot.iris.pipeline.newshader.NewShaderTests.create(NewShaderTests.java:114)
	at net.coderbot.iris.pipeline.newshader.NewWorldRenderingPipeline.createShader(NewWorldRenderingPipeline.java:381)
	at net.coderbot.iris.pipeline.newshader.NewWorldRenderingPipeline.createShader(NewWorldRenderingPipeline.java:374)
	at net.coderbot.iris.pipeline.newshader.NewWorldRenderingPipeline.<init>(NewWorldRenderingPipeline.java:293)
	at net.coderbot.iris.Iris.createPipeline(Iris.java:427)
	at net.coderbot.iris.pipeline.PipelineManager.preparePipeline(PipelineManager.java:32)
	at net.minecraft.class_761.handler$zbp000$iris$beginLevelRender(class_761.java:5450)
	at net.minecraft.class_761.method_22710(class_761.java:1240)
	at net.minecraft.class_757.method_3188(class_757.java:1031)
	at net.minecraft.class_757.method_3192(class_757.java:811)
	at net.minecraft.class_310.method_1523(class_310.java:1117)
	at net.minecraft.class_310.method_1514(class_310.java:733)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.io.IOException: Couldn't compile fragment program (<iris shaderpack shaders>, gbuffers_sky_basic) : 0(3159) : error C1503: undefined variable "MC_RENDER_STAGE_STARS"
	at net.minecraft.class_281.method_34416(class_281.java:72)
	at net.minecraft.class_281.method_1283(class_281.java:54)
	at net.minecraft.class_5944.method_34579(class_5944.java:241)
	at net.minecraft.class_5944.<init>(class_5944.java:183)
	... 21 more
23:05:20.690
Unable to play unknown soundEvent: minecraft:soundtrack.game
```

**Comment by MrX13415** [comment]
This seems to be related to: https://github.com/IrisShaders/Iris/issues/1014

**Comment by IMS212** [comment]
Fixed compiling in 1.2, but there are still other issues.

**Comment by coderbot16** [comment]
Marking this particular issue as fixed for 1.2.5 since we fixed the big issue with stone. The specific issue with entity blurring will be tracked in https://github.com/IrisShaders/Iris/issues/1446.
